crates/ironrdp-web/src/session.rs

```diff
@@ -210,12 +212,22 @@ impl iron_remote_desktop::SessionBuilder for SessionBuilder {
         self.clone()
     }
 
+    /// Disable Network Level Authentication (NLA/CredSSP).
+    ///
+    /// This is useful for Azure AD authentication scenarios where NLA should be disabled
+    /// to allow server-side authentication via winlogon.
+    fn disable_nla(&self, disable: bool) -> Self {
+        self.0.borrow_mut().disable_nla = disable;
+        self.clone()
+    }
+
     fn extension(&self, ext: Extension) -> Self {
         iron_remote_desktop::extension_match! {
             match ext;
             |pcb: String| { self.0.borrow_mut().pcb = Some(pcb) };
             |kdc_proxy_url: String| { self.0.borrow_mut().kdc_proxy_url = Some(kdc_proxy_url) };
             |display_control: bool| { self.0.borrow_mut().use_display_control = display_control };
+            |disable_nla: bool| { self.0.borrow_mut().disable_nla = disable_nla };
```

Comment:
suggestion: Another suggestion. I have a strong preference for using positive flags, because it leads to code that is overall easier to understand. Also, we are using `enable_credssp` in `connector::Config`, I think we should stick to that.

```suggestion
            |enable_credssp: bool| { self.0.borrow_mut().enable_credssp = enable_credssp };
```

for the default value:
```rust
enable_credssp: true,
```

and finally
<img width="698" height="120" alt="Image" src="https://github.com/user-attachments/assets/e8689368-3159-4f5e-92b2-d61c7846388e" />

Reply:
Actually... should we rename the extension to enableCredSsp() as well?